Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -5811,6 +5811,22 @@ PitchCorrectionAlgorithm:
     WebCore:
       default: MediaPlayerEnums::PitchCorrectionAlgorithm::BestAllAround
 
+PointerLockEnabled:
+  type: bool
+  status: internal
+  category: dom
+  condition: ENABLE(POINTER_LOCK)
+  humanReadableName: "Pointer Lock API"
+  humanReadableDescription: "Enable the Pointer Lock API (Locking/Unlocking/Events)"
```

Comment:
"(Locking/Unlocking/Events)" doesn't really seem useful to include 😅

Reply:
"Pointer Lock API" is a grab-bag for various different methods on Document/Element, plus a couple of events. But I think that's well known, so yes probably fair to remove the comment.